reflexio/server/api.py

```
@@ -3389,15 +3474,29 @@ async def lifespan(app: FastAPI) -> AsyncIterator[None]:  # noqa: ARG001
         try:
+            if capabilities is not None:
+                ctx = AppContext()
+                for cap in capabilities.capabilities:
+                    await cap.on_startup(ctx)
+                    started_caps.append(cap)
             yield
         finally:
+            for cap in reversed(started_caps):
+                try:
+                    await cap.on_shutdown()
+                except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)